public/js/extraFunctions.js

```diff
@@ -654,6 +654,143 @@ function tableData_closeAllSelect(elmnt) {
 
 }
 
+let initialize_resize_hamburger = function() {
```

Comment:
Avoid the `let` … `= function()` pattern for top-level functions (because it leads to inconsistent behavior w.r.t. semicolon placement and hoisting):
```suggestion
function initialize_resize_hamburger() {
```